tests/specs/scoped.js

```diff
@@ -334,6 +334,42 @@ test('scoped Store .update()', function (t) {
   })
 })
 
+test('scoped Store .update() updates the correct scope', function (t) {
+  t.plan(7)
+
+  var store = new Store('test-db-scoped-update', merge({remote: 'test-db-scoped-update'}, options))
+
+  var testItems = [
+    { foo: 'bar1', id: 'abc123', type: 'test1' },
+    { foo: 'bar2', id: 'def456', type: 'test2' },
+    { foo: 'bar3', id: 'ghi789', type: 'test2' }
+  ]
+
+  store.add(testItems)
+
+  .then(function (items) {
+    return store('test2').update([
+      { id: testItems[1].id, foo: 'bar20' },
+      { id: testItems[0].id, foo: 'bar10' },
+      { id: testItems[2].id, foo: 'bar100' }
```

Comment:
instead of using `testItems[1].id` etc, would it make the test easier to read if we would put in the id strings directly? We could also name them simpler, like `id1`, `id2` and `id3`?